File: Video 2/2_char.cpp
#include<iostream>
using namespace std ;

int main(){
     // Wrong
     // char ch = "T";
     // char name = "Tirth";
     // char name = 'Tirth';



     // Right
     char ch = 'A';

     int Ascii = ch;
     
     cout << ch << "\n"; 

     Ascii = Ascii + 32 ;

     ch = Ascii;

     cout << ch  ;
     return 0;
}

File: Video 2/Python/2_char.py
# Wrong examples (in C++, not valid in Python either):
# ch = "T"         # This is a string, not a character
# name = "Tirth"   # Again, a string
# name = 'Tirth'   # Still a string, not a single character

# Right
ch = 'A'  # single character

# Get ASCII value of character
ascii_value = ord(ch)

print(ch)  # Output: A

ascii_value = ascii_value + 32  # Convert to lowercase by adding 32

ch = chr(ascii_value)  # Convert back to character

print(ch)  # Output: a
